cri/apis/cri/runtime/v1alpha2/api.proto

```diff
@@ -0,0 +1,1281 @@
+// To regenerate api.pb.go run hack/update-generated-runtime.sh
```

Comment:
`cri/apis/cri/v1alpha1` ---> `cri/apis/v1alpha1`
`cri/apis/cri/runtime/v1alpha2` ---> `cri/apis/v1alpha2`

make the directory structure more clear